packages/protocol/migrations/20_double_signing_slasher.ts

```diff
@@ -15,5 +18,12 @@ module.exports = deploymentForCoreContract<DoubleSigningSlasherInstance>(
   web3,
   artifacts,
   CeloContractName.DoubleSigningSlasher,
-  initializeArgs
+  initializeArgs,
+  async (slasher: DoubleSigningSlasherInstance) => {
+    const lockedGold: LockedGoldInstance = await getDeployedProxiedContract<LockedGoldInstance>(
+      'LockedGold',
+      artifacts
+    )
+    await lockedGold.addSlasher(slasher.address)
```

Comment:
This comes from #2347 and we should remove it

Reply:
Current version won't work without this whitelist.